recipes/napari-pdf-reader/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "napari-pdf-reader" %}
+{% set version = "0.0.1a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-pdf-reader-{{ version }}.tar.gz
+  sha256: b2fe4e7bfaf06f96d10e60a5f3c32dcbb0d40a52fc65e4b911571c51a819a3d2
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools-scm
+  run:
+    - napari-plugin-engine >=0.1.4
+    - numpy
+    - pdf2image
+    - pillow
+    - python >=3.7
+
+test:
+  imports:
+    - napari_pdf_reader
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/krentzd/napari-pdf-reader
+  summary: Reader for PDF files
```

Comment:
Maybe this could be reworded a little so it's clear that this is not a PDF Reader (like Acrobat Reader) but specific to the napari project (which I was not familiar with until a moment ago.)